packages/trader/src/Modules/Trading/Containers/purchase.jsx

```diff
@@ -4,6 +4,7 @@ import ObjectUtils                 from 'deriv-shared/utils/object';
 import PurchaseFieldset            from 'Modules/Trading/Components/Elements/purchase-fieldset.jsx';
 import { getContractTypePosition } from 'Constants/contract';
 import { connect }                 from 'Stores/connect';
+import { measurePerformance }      from '../../../Services/perfomance-checker';
```

Comment:
the import can directly be from `Services/` route, no need to cd to it

Reply:
sure, thanks